src/extensions/mod.rs

```diff
@@ -99,6 +115,15 @@ pub trait RelmListBoxExt {
 }
 
 impl RelmListBoxExt for gtk::ListBox {
+    fn rows(&self) -> Vec<gtk::ListBoxRow> {
+        iter_children(self.as_ref())
+            .map(|widget| {
+                widget
+                    .downcast::<gtk::ListBoxRow>()
+                    .expect("The child of `ListBox` is not a `ListBoxRow`.")
+            })
+            .collect()
+    }
     fn index_of_child(&self, widget: &impl AsRef<gtk::Widget>) -> Option<i32> {
         if let Some(row) = self.row_of_child(widget) {
             return Some(row.index());
```

Comment:
Can't you do something like `self.row_of_child(widget).map(|row| row.index())` ?

Reply:
Sure.